### DISCUSSION TITLE
Should I keep the root eslintrc?

### DISCUSSION THREAD

**Discussion by angel-luis** [vercel/turborepo#8417] [Help] [answered]
I don't get the point of having by default in Turborepo a `.eslintrc.js` file at root level. This file is ignoring the `apps` and `packages` folder. The rest of the files in a Turborepo is just the `turbo.json` and the `package.json`. Also, this file extends the TS parser and the Node compiler options. 

In what cases could this be beneficial? According to the documentation:

> The only dependencies that belong in the workspace root are tools for managing the repository whereas dependencies for building applications and libraries are installed in their respective packages. Some examples of dependencies that make sense to install in the root are [turbo](https://www.npmjs.com/package/turbo), [husky](https://www.npmjs.com/package/husky), or [lint-staged](https://www.npmjs.com/package/lint-staged).

So I think there is no reason to have this file when creating a new Turborepo.

**Comment by anthonyshew** [comment, marked as answer]
Fixing in https://github.com/vercel/turbo/pull/8423
